frontend/src/features/public-form/components/FormEndPage/FormEndPageContainer.tsx

```diff
@@ -42,7 +56,9 @@ export const FormEndPageContainer = (): JSX.Element | null => {
         formTitle={form.title}
         endPage={form.endPage}
         isFeedbackSubmitted={isFeedbackSubmitted}
-        handleSubmitFeedback={handleSubmitFeedback}
+        handleSubmitFeedback={
+          isPreview ? handleSubmitFeedbackPreview : handleSubmitFeedback
+        }
```

Comment:
nit: should we just call the preview function in `handleSubmitFeedback` directly